msprime/likelihood.py

```diff
@@ -60,9 +60,9 @@ def unnormalised_log_mutation_likelihood(arg, theta):
     return ret
 
 
-def log_arg_likelihood(arg, recombination_rate, Ne=0.25):
+def log_arg_likelihood(arg, recombination_rate, Ne=0.5):
```

Comment:
We'll need to change the default to 1 before release --- is there much work in changing the tests?

Reply:
Nah, it shouldn't take me too long. I'll probably not have a chance to get to it this week, but will get it done shortly thereafter.

Reply:
Great, let's get that in with this PR so. Thanks.

Reply:
Having looked at it, it turns out that the tests don't use the default value of Ne. As of the latest commit, they all feed in an explicit argument of 0.5, so we can change the default without affecting tests in any way. Would that be ok, or do you think we need a more comprehensive overhaul?

Reply:
No, great, let's just change the default and write a couple of quick tests to check the default is what we think it is.